src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$card-bg: #FFF;
+$card-reviews-text: #060B35;
+$card-price-text: #060B35;
+$card-title-text: #060B35;
```

Comment:
Why are you creating multiple variables with the same value? If only one variable can be used everywhere
```suggestion
$card-reviews-text: #060B35;
```